sdk/inc/azure/iot/az_iot_adu_client.h

```diff
@@ -44,60 +45,31 @@
  */
 #define AZ_IOT_ADU_CLIENT_PROPERTIES_COMPONENT_NAME "deviceUpdate"
 
-/**
- * @brief ADU Service Response (Accepted)
- */
-#define AZ_IOT_ADU_CLIENT_REQUEST_ACCEPTED 200
-
-/**
- * @brief ADU Service Response (Rejected)
- */
-#define AZ_IOT_ADU_CLIENT_REQUEST_REJECTED 406
-
-/**
- * @brief ADU Service Action (Apply)
- */
-#define AZ_IOT_ADU_CLIENT_SERVICE_ACTION_APPLY_DEPLOYMENT 3
-
-/**
- * @brief ADU Service Action (Cancel)
- */
-#define AZ_IOT_ADU_CLIENT_SERVICE_ACTION_CANCEL 255
-
-/**
- * @brief ADU Agent State (Idle)
- */
-#define AZ_IOT_ADU_CLIENT_AGENT_STATE_IDLE 0
-
-/**
- * @brief ADU Agent State (In Progress)
- */
-#define AZ_IOT_ADU_CLIENT_AGENT_STATE_DEPLOYMENT_IN_PROGRESS 6
-
-/**
- * @brief ADU Agent State (Failed)
- */
-#define AZ_IOT_ADU_CLIENT_AGENT_STATE_FAILED 255
-
-/**
- * @brief  Maximum Number of Files Handled by this ADU Agent (Number of URLs)
- */
-#define AZ_IOT_ADU_CLIENT_MAX_FILE_URL_COUNT 10
-
-/**
- * @brief  Maximum Number of Files Handled by this ADU Agent (Steps)
- */
-#define AZ_IOT_ADU_CLIENT_MAX_INSTRUCTIONS_STEPS 10
+enum
+{
+  /// ADU Service Response (Accepted)
+  AZ_IOT_ADU_CLIENT_REQUEST_ACCEPTED = 200,
+  /// ADU Service Response (Rejected)
+  AZ_IOT_ADU_CLIENT_REQUEST_REJECTED = 406
+};
```

Comment:
these and below can debatably be typed.

Reply:
For this one, I would say let's type it.

Reply:
I think all of these should be named so that the public API is easier to follow w/o (or with limited) documentation searches.